src/cq_cam/common.py

```diff
@@ -1,12 +1,270 @@
-from enum import Enum
+from enum import Enum, auto
+
+"""
```

Comment:
If you move this string literal above the import it becomes a module docstring ( https://peps.python.org/pep-0257/#what-is-a-docstring ). Might be preferable?

Reply:
I was wondering how to do that. Thanks!